### PR TITLE
✨ Add player account system

### PR DESCRIPTION
**Pull request by zakio-dev**

### CODE REVIEW

File: src/class/NCSCharacter/server/functions/removeAccount.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---@param amount number
+function NCSCharacter.removeAccount(self, _, accountName, amount)
+    if (not self) then
+        return NCS:die("Character not found.")
+    end
+
+    if (not self.accounts[accountName]) then
+        return NCS:die("Account not found.")
+    end
+
```
Comment (CsokiHUN) [inline src/class/NCSCharacter/server/functions/removeAccount.lua:14]:
```suggestion
    amount = (amount or 0)
```

File: src/class/NCSCharacter/server/functions/saveAccounts.lua [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---savePosition
+---@public
+---@param self NCSCharacter
+function NCSCharacter.saveAccounts(self)
+    if (not (self.skin)) then
```
Comment (CsokiHUN) [inline src/class/NCSCharacter/server/functions/saveAccounts.lua:5]:
```suggestion
    if (not (self.accounts)) then
```

**Review by CsokiHUN — CHANGES_REQUESTED** [review]

File: src/class/NCSCharacter/server/functions/removeAccount.lua (hunk 1 above)
Comment (zakio-dev) [reply src/class/NCSCharacter/server/functions/removeAccount.lua:14]: If i do that account can be negative

Comment (CsokiHUN) [reply src/class/NCSCharacter/server/functions/removeAccount.lua:14]: no, unless you accidentally have no value for the amount variable, you won't get an error.

Comment (zakio-dev) [reply src/class/NCSCharacter/server/functions/removeAccount.lua]: Ah, yes i'm stupid

**Comment by MoskalykA** [comment]
After a short discussion with @PABLO-1610, we find the idea interesting but not necessary in the core. Thanks anyway for your contribution.

**Pull request closed without merging**